geode-core/src/integrationTest/java/org/apache/geode/cache/client/internal/QueueManagerJUnitTest.java

```diff
@@ -312,6 +378,21 @@ private static void assertPortEquals(int[] expected, Iterable<Connection> actual
     assertThat(actualPorts).isEqualTo(expectedPorts);
   }
 
+  private class RecoveryDummyPool extends DummyPool {
```

Comment:
Let's avoid the term "dummy" please. `TestPool` or `FakePool` would be acceptable.

Reply:
I agree. I was following the existing naming which I am not a fan of either.